docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+MAINTAINER svanoort <samvanoort@gmail.com>
+
+
+# Dependencies plus vim for convenience, wget if you're trying to get fancy
+RUN apt-get update \
+  && apt-get install -y make unzip devscripts debhelper rpm expect createrepo ruby maven openjdk-7-jre \
+  && apt-get install -y vim git \
+  && apt-get install -y wget \ 
```

Comment:
Maybe makes sense to ensure that all SSH and font packages are installed.
Also makes sense to install dot/graphviz for the depencdency graph visualization. Missing packages may cause test failures (if we have such tests).